Crab Game/Crab Game/SettingsViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SettingsViewController.swift
+//  Crab Game
+//
+//  Created by Ian Becker on 5/18/20.
+//  Copyright © 2020 Ian Becker. All rights reserved.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+    
+    @IBAction func toggleRoll(_ sender: UISwitch) {
+        // If switch is on then roll
```

Comment:
This actually only needs one line of code. `Settings.shared.shouldRoll = sender.isOn`